Cookbook/Technical-Documents/Development-Process.md

```diff
@@ -0,0 +1,61 @@
+# Development Process
+
+Babylon follows the [SSDLC](https://engineering.ops.babylontech.co.uk/docs/standards-ssdlc) when developing our software. To make the process easier, we use a few services to link code changes to tickets.
+
+There are 3 services we use daily:
+
+- Jira: this is where the development begins. Once a ticket has all the necessary details, dev can pick it up and implement it. <!-- TODO: [CNSMR-3230] link to Jira article -->
+- GitHub: used for hosting our project
+- CircleCI: our continuous integration service to run tests when integrating PRs <!-- TODO: [CNSMR-xxxx] link to CircleCI article -->
+
+## Jira 🎫
+
+- Everything we do must have a ticket
+- The first step is to assign the ticket to the engineer. Make sure it has all details and designs attached (if applicable)
+- The ticket is moved to `In progress` (this might be automatically done by the bot when branch published, if configured)
```

Comment:
```suggestion
- The ticket will be moved to `In progress` once development is ready to start (this might be automatically done by the bot when branch published, if configured).
```